pkg/promclient/promclient.go

```diff
@@ -17,6 +21,12 @@ import (
 	"strings"
 	"time"
 
+	"github.com/go-kit/kit/log/level"
+
```

Comment:
could you remove the blank lines manually and the goimports will fix it :)

Reply:
):